### ISSUE TITLE
Path Page Implementation

### ISSUE THREAD

**Issue by meltemarsl** [bounswe/2021SpringGroup11#225]
Path page implementation should be done until 11.12.2021- 10.00 since I will be adding functionalities to it.
Another branch should be created and used for this work. (name suggestion: "path")
The design was not completed but you can utilize this:
![image](https://user-images.githubusercontent.com/44238703/145453422-1e26e465-849d-4acf-bae5-1be2bcc6559c.png)

but you should use the format drawn on this meeting:

https://github.com/bounswe/2021SpringGroup11/wiki/Meeting-17.1

**Comment by bekirtatar** [comment]
This issue is transferred to another issue.
